src/test/java/LoginCourierTest.java

```diff
@@ -0,0 +1,67 @@
+import io.qameta.allure.junit4.DisplayName;
+import jdk.jfr.Description;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class LoginCourierTest {
+    CourierClient courierClient;
+    private int courierId;
+    Courier courier = new Courier("Alex3000", "123456", "Alex3000");
+
+    @Before
+    public void setUp(){
+        courierClient = new CourierClient();
+    }
+
+    @Test
+    @DisplayName("Проверка получения Id курьера.Успешная авторизация.")
+    @Description("Проверка получения Id курьера после успешной авторизации.")
+    public void getIdAfterLoginTest(){
+        courierId = courierClient.loginAndGetId(CourierCredentials.getCourierCredentials(courier));
+
+        assertThat("Courier id is null",courierId,is(not(0)));
+    }
+
+    @Test
+    @DisplayName("Негативный сценарий авторизации. Авторизация с неверным паролем")
+    @Description("Проверка сообщения об ошибке при авторизации с неверным паролем и существующем логином")
+    public void loginWithWrongPasswordTest(){
+        Courier courier = new Courier("Alex3000", "1234567", "Alex3000");
+        String actualMessage = courierClient.loginWithWrongPassword(CourierCredentials.getCourierCredentials(courier));
+
+        assertThat("Error message test of loginWithWrongPasswordTest was changed","Учетная запись не найдена",equalTo(actualMessage));
+    }
+
+   @Test
+   @DisplayName("Негативный сценарий авторизации. Авторизация без пароля.")
+   @Description("Проверка сообщения об ошибке при авторизации с существующем логином, но без поля пароль")
+    public void loginWithoutObligatoryFieldTest(){
```

Comment:
Твой тест падает с очень не хорошей ошибкой:
_Cannot invoke the path method because no content-type was present in the response and no default parser has been set.
You can specify a default parser using e.g.:
RestAssured.defaultParser = Parser.JSON;_
Со стороны это выглядит как косяк тестов, а не косяк сервиса. А это не так. Такая ситуация происходит из-за того, что наш сервер виснет не отдает тело ответа, но зато он отдает статус код. Если проверка на статус код делалась бы раньше проверки body, то тест бы падал с красивой ошибкой статус кода и вопросов бы не было. 
Поэтому я всегда рекомендую первым делом проверять статус код, а потом уже все остальное.